openedx/core/djangoapps/safe_sessions/middleware.py

```diff
@@ -301,12 +300,14 @@ def process_request(self, request):
         final verification before sending the response (in
         process_response).
         """
-        # 2021-10-29: Temporary debugging attr to answer the question
+        # 2021-12-01: Temporary debugging attr to answer the question
         # "are browsers sometimes sending in multiple session
-        # cookies?" We've observed behavior that might be consistent
-        # with this, perhaps due to an additional cookie set on the
-        # wrong domain, and assuming that the cookies *occasionally*
-        # are sent in a different order. -- timmc
+        # cookies?" Answer: Yes, this sometimes happens, although it
+        # does not appear to cause user mismatches. (There was a theory
+        # that multiple cookies might sometimes be sent in a varying
+        # order.) We may still want to have the ability to monitor this
+        # oddity, but as far as I can tell, it is not essential to the
```

Comment:
Consider using “we” and dropping your name, since it has been reviewed.

Reply:
Makes sense. :-) And it's no longer just speculative attribute-adding -- we actually have some data now.